vendor/bat-native-ads/src/bat/ads/internal/security/crypto_util.cc

```diff
@@ -162,42 +120,83 @@ std::vector<uint8_t> Sha256Hash(const std::string& value) {
   return sha256;
 }
 
-bool Verify(const ConfirmationInfo& confirmation) {
-  std::string credential;
-  base::Base64Decode(confirmation.credential, &credential);
-
-  base::Optional<base::Value> value = base::JSONReader::Read(credential);
-  if (!value || !value->is_dict()) {
-    return false;
+KeyPairInfo GenerateSignKeyPairFromSeed(const std::vector<uint8_t>& seed) {
+  KeyPairInfo key_pair;
+  if (seed.empty()) {
+    return key_pair;
   }
 
-  base::DictionaryValue* dictionary = nullptr;
-  if (!value->GetAsDictionary(&dictionary)) {
-    return false;
-  }
+  std::vector<uint8_t> secret_key = seed;
```

Comment:
Im not sure why we need this `seed` value? The bytes are just used to initialize the vector that `crypto_sign_keypair` uses to fill in its generated keys. You mentioned this to me in slack and I want to relay this here. I don't like this and it should be removed.

Reply:
https://nacl.cr.yp.to/sign.html

Reply:
Thank you for creating the ticket. There is no security concern in not-resolving now so I am fine with it.